### ISSUE TITLE
Cache playlist to avoid interface loading time

### ISSUE THREAD

**Issue by gagarine** [jpochyla/psst#316]
When you click on a playlist, you have to wait about 1s for the interface to load while psst do a network request. This is not the case with "Albums" for example.

I think playlist could be cached (and perhaps all fetched in the background at launch). When you click tracks list could then be instantly display. The network request can be done in the background once you opened a playlist and only if the playlist changed, then the interface could be updated.

This would give a

**Comment by jacksongoode** [comment]
This is a nice idea, how would you imagine the playlist gets updated? Just a hard refresh of the view once the request for the playlist is complete?

**Comment by gagarine** [comment]
Yes. Hard refresh is certainly fine. One things would be to keep selected the track if the user had the time to select one before refresh.
